src/sidebar/components/NotebookView.js

```diff
@@ -34,6 +34,13 @@ function NotebookView({ loadAnnotationsService }) {
 
   const groupName = focusedGroup?.name ?? '…';
 
+  // Get the ID of the group to fetch annotations from. Once groups are fetched
```

Comment:
"Once groups are fetched this is this same as the focused group ID" doesn't immediately make sense to me.

Reply:
I've reworded this comment to hopefully make it clearer.